crates/wash-cli/src/bin/wash.rs

```diff
@@ -511,12 +531,22 @@ async fn main() {
         std::process::exit(0);
     };
 
+    if cli.version {
+        println!("{}", version(cli.output));
+        std::process::exit(0);
+    }
+
+    let cli_command = cli.command.unwrap_or_else(|| {
+        eprintln!("{}", command.render_help());
+        std::process::exit(2);
+    });
```

Comment:
Quick check, does this print the help and exit with code `2` if I specifically type `wash --help`?

Reply:
edit: I checked! If you provide `--help` it still exits with code 0

Reply:
If you run `wash` without any arguments it currently exits with code 2. That was the convention I followed but happy to change.